scripts/publishRelease.sh

```diff
@@ -4,6 +4,7 @@
 TAG=$TAG
 
 RELEASE_SYNTAX='^[0-9]+\.[0-9]+\.[0-9]+$'
+MASTER_PRERELEASE_SYNTAX='^[0-9]+\.[0-9]+\.[0-9]+-master\..+$'
```

Comment:
This is kinda pointless as we can just simply check if tag equals master?

Reply:
the tag will be actually something like `2.6.0-master.967.57a7482`, then dist-tag `master` should point to it

Reply:
@jhellar now that you're using the `dev` tag on npm. Maybe we should change this to use the word `dev` instead of master also?

Reply:
@darahayes you mean just for the version/variable name, or should it be changed everywhere - not mention master but dev?

Reply:
Yeah I mean the actual value so you will have something like `2.6.0-dev.967.57a7482`

Reply:
ok, makes sense, will change it